scripts/install_platform.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+CURRENT_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)
+source "${CURRENT_DIR}/formats.sh"
+
+if [ -z "${SC_MACHINE_PATH}" ];
```

Comment:
I am not entirely comfortable checking just one variable rather than all the variables that we're using in a script. We could check all and if 1 or more are absent we could source the `env_vars.sh`.
There is a challenge though:
1. Lower-level repos (such as this one) can re-export variables with a hardcoded value, so it might override variables with an unexpected value
2. The error message will be confusing, so perhaps we can just fail the script if one of the variables is present, but the other ones are not? I'm not sure how exactly can we track that `env_vars.sh` was already sourced. We could a special "marker" variable such as `export SC_ENV=1`, but that doesn't look elegant and can lead to other issues that are not apparent at the moment.